lms/templates/discussion/_underscore_templates.html

```diff
@@ -1,607 +1,14 @@
-                <span class="sr">${sr_label}</span>
-                <span class="action-label" aria-hidden="true">
-                    <span class="label-unchecked">${unchecked_label}</span>
-                    <span class="label-checked">${checked_label}</span>
-                </span>
-                <span class="action-icon">
-                  <i class="icon fa fa-${icon}"></i>
-                </span>
-            </a>
-        </li>
-    </script>
-</%def>
-
-${secondaryStateAction("report", "flag", _("Report abuse"), _("Report"), _("Unreport"))}
-${secondaryStateAction("pin", "thumb-tack", _("Pin"), _("Pin"), _("Unpin"))}
-${secondaryStateAction("close", "lock", _("Close"), _("Close"), _("Open"))}
-
-<%def name="secondaryAction(action_class, icon, label)">
-    <script type="text/template" id="forum-action-${action_class}">
-        <li class="actions-item">
-            <a href="javascript:void(0)" class="action-list-item action-${action_class}" role="button">
-                <span class="action-label">${label}</span>
-                <span class="action-icon"><i class="icon fa fa-${icon}"></i></span>
-            </a>
-        </li>
-    </script>
-</%def>
-
-${secondaryAction("edit", "pencil", _("Edit"))}
-${secondaryAction("delete", "remove", _("Delete"))}
-
-<script type="text/template" id="forum-actions">
-    <ul class="${"<%= contentType %>"}-actions-list">
-        ${"<% _.each(primaryActions, function(action) { print(_.template($('#forum-action-' + action).html(), {})) }) %>"}
-        <li class="actions-item is-visible">
-            <div class="more-wrapper">
-                <a href="javascript:void(0)" class="action-button action-more" role="button" aria-haspopup="true" aria-controls="action-menu-${"<%= contentId %>"}">
-                    <span class="action-label">${_("More")}</span>
-                    <span class="action-icon"><i class="icon fa fa-ellipsis-h"></i></span>
-                </a>
-                <div class="actions-dropdown" id="action-menu-${"<%= contentType %>"}" aria-expanded="false">
-                  <ul class="actions-dropdown-list">
-                    ${"<% _.each(secondaryActions, function(action) { print(_.template($('#forum-action-' + action).html(), {})) }) %>"}
-                  </ul>
-                </div>
-            </div>
-        </li>
-    </ul>
-</script>
-
-<script aria-hidden="true" type="text/template" id="post-user-display-template">
-    ${"<% if (username) { %>"}
-    <a href="${'<%- user_url %>'}" class="username">${'<%- username %>'}</a>
-        ${"<% if (is_community_ta) { %>"}
-        <span class="user-label-community-ta">${_("Community TA")}</span>
-        ${"<% } else if (is_staff) { %>"}
-        <span class="user-label-staff">${_("Staff")}</span>
-        ${"<% } %>"}
-    ${"<% } else { %>"}
-    ${_('anonymous') | h}
-    ${"<% } %>"}
+## same, but without trailing "-template" in script ID
```

Comment:
I don't think it's a good idea to roll that change into this pull request -- the PR is big enough as it is. That can go into a follow-up PR, if we want.